java/ql/test/library-tests/pathsanitizer/Test.java

```diff
@@ -279,7 +279,7 @@ public void dotDotCheckGuard() throws Exception {
     }
 
     private void blockListGuardValidation(String path) throws Exception {
-        if (path.contains("..") || !path.startsWith("/data"))
+        if (path.contains("..") || path.startsWith("/data"))
```

Comment:
Intentional change?

Reply:
Yes, a blocklist guard should check for forbidden paths + path normalization/check for `..`. So it was wrong before this change — of course, the Java test should've failed before, I still need to figure out why it didn't.